apps/homework/models.py

```diff
@@ -187,9 +187,12 @@ def get_total_score(self):
     def calculate_grade(self):
         total, total_possible = self.get_total_score_total_possible()
         self.text_grade = f"{total}/{total_possible}"
-        if total_possible != 0 and total != 0:
+        if total_possible != 0 and total is not None:
             self.overall_grade = total/total_possible
```

Comment:
need space around operator (not in this PR I know but important!) :)

Reply:
Will do. Is this purely a style thing?

Reply:
✅

Reply:
i think that's pep8, but in some cases you _don't_ want spaces around operators, interesting.. https://www.python.org/dev/peps/pep-0008/#other-recommendations